frontstage/views/sign_in/sign_in.py

```diff
@@ -104,6 +104,7 @@ def login():  # noqa: C901
             expires=redis_session.get_expires_in(),
             secure=secure,
             httponly=secure,
+            samesite="strict",
```

Comment:
why not secure?

Reply:
You can only set `samesite` to `None`, `Lax`, or `Strict`. There is no `secure` option. Detailed here: https://blog.heroku.com/chrome-changes-samesite-cookie#how-is-chrome-protecting-users-against-csrf-attacks